fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1155,6 +1155,18 @@ create_stmt ::=
         RESULT = new CreateTableStmt(ifNotExists, isExternal, name, columns, indexes, engineName, keys, partition,
         distribution, tblProperties, extProperties, tableComment, index);
     :}
+    | KW_CREATE opt_external:isExternal KW_TABLE opt_if_not_exists:ifNotExists table_name:name
+        opt_col_list:columns
```

Comment:
Also support column_definition_list?

Reply:
support the simple case first and look at the requirements